src/main/java/org/micks/champmaker/championships/ChampionshipService.java

```diff
@@ -53,13 +53,20 @@ public void createChampionship(ChampionshipDTO championshipDTO) {
     public void shuffleTeams(long champId) {
         List<RegisterEntity> registeredTeamList = registerRepository.findByChampId(champId);
         Random random = new Random();
+        int counterA =0;
+        int counterB =0;
```

Comment:
(nice to have) please use ctrl+alt+L to fix styling, like adding spaces after =
```suggestion
        int counterA = 0;
        int counterB = 0;
```

Reply:
Fixed